packages/terra-image/src/Image.jsx

```diff
@@ -49,20 +58,27 @@ const propTypes = {
    * Function to be executed when the image load errors.
    */
   onError: PropTypes.func,
+  /**
+  * Sets the `object-fit` style of the image from the following values; `cover`, `contain`, `fill`, `scale-down`, `none`.
+  */
+  imageFit: PropTypes.oneOf(['cover', 'contain', 'fill', 'scale-down', 'none']),
 };
 
 /* eslint-disable react/default-props-match-prop-types */
 const defaultProps = {
   variant: 'default',
   isFluid: false,
   alt: ' ',
+  imageFit: 'fill',
 };
 /* eslint-enable react/default-props-match-prop-types */
 
 class Image extends React.Component {
   constructor(props) {
     super(props);
 
+    this.myRef = React.createRef();
```

Comment:
This could have a better name, maybe?

Reply:
Fixed here https://github.com/cerner/terra-core/pull/2494/commits/0127dec949032aa00c86bb77ebdeb92d3d695832.